<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-26T21:38:28 IST" name="Default suite" finished-at="2024-05-26T21:38:40 IST" duration-ms="12415">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.Runtest@66982506]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-05-26T21:38:28 IST" name="Default test" finished-at="2024-05-26T21:38:40 IST" duration-ms="12415">
      <class name="testRunner.Runtest">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:testRunner.Runtest@66982506]" started-at="2024-05-26T21:38:28 IST" name="setUpClass" finished-at="2024-05-26T21:38:28 IST" duration-ms="168" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4110765e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.Runtest@66982506]" started-at="2024-05-26T21:38:28 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-05-26T21:38:40 IST" duration-ms="12192" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Search customer by Firstname and Lastname"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Customer"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at stepDefintions.Steps.user_should_find_customer_name_info_in_search_list(Steps.java:172)
at âœ½.user should find customer name info in search list(file:///C:/Users/sande/eclipse-workspace/CucumberPracticeYTpavan/src/test/java/java/Feature/Customer.feature:31)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:testRunner.Runtest@66982506]" started-at="2024-05-26T21:38:40 IST" name="tearDownClass" finished-at="2024-05-26T21:38:40 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- testRunner.Runtest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
